src/installer/Directory.Build.props

```diff
@@ -138,175 +115,22 @@
-        <TargetsUnix>true</TargetsUnix>
-      </PropertyGroup>
-    </When>
-    <When Condition="$(OutputRid.StartsWith('illumos'))">
-      <PropertyGroup>
-        <Targetsillumos>true</Targetsillumos>
-        <TargetsUnix>true</TargetsUnix>
-      </PropertyGroup>
-    </When>
-    <When Condition="$(OutputRid.StartsWith('solaris'))">
-      <PropertyGroup>
-        <TargetsSolaris>true</TargetsSolaris>
-        <TargetsUnix>true</TargetsUnix>
-      </PropertyGroup>
-    </When>
-    <Otherwise>
-      <PropertyGroup>
-        <TargetsLinux>true</TargetsLinux>
-        <TargetsUnix>true</TargetsUnix>
-      </PropertyGroup>
-      <PropertyGroup Condition="'$(PortableBuild)' == 'true'">
-        <HostMachineRidTargetsDebianPackages Condition="
-          $(HostRuntimeIdentifier.StartsWith('debian')) or
-          $(HostRuntimeIdentifier.StartsWith('ubuntu')) or
-          $(HostRuntimeIdentifier.StartsWith('linuxmint'))">true</HostMachineRidTargetsDebianPackages>
-        <!-- If the build machine isn't known to be Debian-based, try to build RPM packages. -->
-        <HostMachineRidTargetsRpmPackages Condition="'$(HostMachineRidTargetsDebianPackages)' != 'true'">true</HostMachineRidTargetsRpmPackages>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
 
   <PropertyGroup>
     <CompressedFileExtension Condition="'$(TargetOS)' == 'windows'">.zip</CompressedFileExtension>
     <CompressedFileExtension Condition="'$(TargetOS)' != 'windows'">.tar.gz</CompressedFileExtension>
     <InstallerExtension Condition="'$(TargetOS)' == 'windows'">.msi</InstallerExtension>
     <InstallerExtension Condition="'$(TargetOS)' == 'OSX'">.pkg</InstallerExtension>
-    <InstallerExtension Condition="'$(TargetsDebian)' == 'true' or '$(TargetsUbuntu)' == 'true' or '$(TargetsLinuxMint)' == 'true' or '$(HostMachineRidTargetsDebianPackages)' == 'true'">.deb</InstallerExtension>
-    <InstallerExtension Condition="'$(TargetsRhel)' == 'true' or '$(TargetsCentos)' == 'true' or '$(TargetsOpensuse)' == 'true' or '$(TargetsFedora)' == 'true' or '$(TargetsOracle)' == 'true' or '$(TargetsSles)' == 'true' or '$(HostMachineRidTargetsRpmPackages)' == 'true'">.rpm</InstallerExtension>
+    <InstallerExtension Condition="$(OutputRid.StartsWith('debian')) or $(OutputRid.StartsWith('ubuntu')) or $(OutputRid.StartsWith('linuxmint')) or '$(HostMachineRidTargetsDebianPackages)' == 'true'">.deb</InstallerExtension>
+    <InstallerExtension Condition="$(OutputRid.StartsWith('rhel')) or $(OutputRid.StartsWith('centos')) or $(OutputRid.StartsWith('opensuse')) or $(OutputRid.StartsWith('fedora')) or $(OutputRid.StartsWith('oracle')) or $(OutputRid.StartsWith('sles')) or '$(HostMachineRidTargetsRpmPackages)' == 'true'">.rpm</InstallerExtension>
     <CombinedInstallerExtension Condition="'$(TargetOS)' == 'windows'">.exe</CombinedInstallerExtension>
     <CombinedInstallerExtension Condition="'$(TargetOS)' != 'windows'">$(InstallerExtension)</CombinedInstallerExtension>
```

Comment:
Same as above with this block. These properties are handled within the new tooling so we don't need to set them here.

Reply:
These properties were used in an unused tarter `GetInstallerLocations`. I have deleted it as well.